feature/login.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Login no aplicativo
+    As a usuário do aplicativo
+    I want to entrar no aplicativo com meu e-mail e senha cadastrado
+    So that eu tenha acesso às funcionalidades do sistema que são acessíveis somente depois do login
+
+Scenario: Login correto
+    Given o usuário está na página de "login"
+    And existe um usuário cadastrado
+    When o usuário preenche os campos de cpf e senha corretamente
+    Then o usuário é encaminhado para a tela inicial
+
+Scenario: Login admin
+    Given o usuário está na página de "login"
+    And existe um usuário cadastrado
+    When o usuário preenche os campos de cpf e senha com as credenciais de administrador corretamente
+    Then o admin é encaminhado para a pagina do administrador
+
+Scenario: Login incorreto
+    Given o usuário está na página de "login"
+    And existe um usuário cadastrado
+    When eu tento acessar o aplicatico com o cpf "11111111111"
+    And o campo da senha é preenchido por "123456"
+    Then eu recebo uma mensagem  "Credenciais incorretas"
+    And eu continuo na página de login
+
+
+Scenario: Esqueceu a senha e preenche o campo de recuperar com o código certo
```

Comment:
You need to break this test into two other. First, you'll need a test to make sure the user can go "esqueci a senha" page and then you make another test where the user is already at the "esqueci a senha" page an from there he'll try to recover his password. 

You need to specify what is the code that you'll send in the email so that you can test with this specific code.

Remeber that the user must be identified by the CPF, not the username.

Reply:
Remeber to put the pages as parameters.

Reply:
Remember that at the beggining you don't know who is the user.